F/FreeTypeAbstraction/Compat.toml

```diff
@@ -1,6 +1,13 @@
-["0.4"]
+["0-0.4"]
 ColorVectorSpace = "0.6-0"
 Colors = "0"
```

Comment:
We should cap these at some point, but could be done in a more wholesale manner with https://github.com/bcbi/RetroCap.jl